src/Reducers/books.js

```diff
@@ -0,0 +1,20 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../Actions';
+
+const initstate = { books: [{ Id: Math.random(), Title: 'Harry Potter & The Prisoners of Askaban', Category: 'Kids' }, { Id: Math.random(), Title: 'Shoe Dog', Category: 'Biography' }, { Id: Math.random(), Title: 'The Great Gatsby', Category: 'Action' }] };
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Harry Potter & The ..', category: 'Kids'}
```

Reply:
Math.random() has been asked in the pathwright module